### PR TITLE
data: imaaquibali

### PR DESCRIPTION
**Pull request by imaaquibali**
## Changes proposed

Added profile 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Add this profile as soon as possible

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4961"></a>

### CODE REVIEW

**Review by himanshu1221 — CHANGES_REQUESTED** [review]
Hey @imaaquibali please put your file in `data` folder

**Comment by imaaquibali** [comment]
> Hey @imaaquibali please put your file in `data` folder

Done. Can you please check it know.

**Review by himanshu1221 — APPROVED** [review]
**LGTM :+1:**

**Review by devvsakib — APPROVED** [review]
Amazing

File: data/imaaquibali.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+  "name": "Aaquib Ali",
+  "type": "personal",
+  "bio": "DevRel | Founder of **The VerSiton**| Mentor | Open-Source Contributor | **Content Creator** |",
+
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Content Creator"
```
Comment (Pradumnasaraf) [inline data/imaaquibali.json:9]:
```suggestion
    "Content Creator",
```

**Review by Pradumnasaraf — DISMISSED** [review]
Hey,

There is a missing `,` comma. Fix that

**Comment by SaraJaoude** [comment]
looks good

**Pull request merged**